<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page2</class>
 <widget class="QMainWindow" name="page2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>948</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>410</y>
      <width>531</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>show captcha</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>620</y>
      <width>531</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>check it</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>840</y>
      <width>531</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign in</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>901</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(122, 122, 122);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>531</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Email or phone</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>531</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>50</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;0&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>491</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>0</y>
      <width>821</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;MV Boli&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Sign in&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>531</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>750</y>
      <width>531</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>verify your email address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Get code</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Check</string>
     </property>
    </widget>
   </widget>
   <zorder>label_7</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>label_6</zorder>
   <zorder>label_3</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>groupBox_6</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
